test/functional/wallet_basic.py

```diff
@@ -504,15 +504,58 @@ def run_test(self):
                                 {"address": address_to_import},
                                 {"spendable": False})
 
-            # 5. Import private key of the previously imported address on node1
+            # 5. Remove the address from node1
+            self.nodes[1].removeaddress(address_to_import, False, False)
+
```

Comment:
Would be good to validate here that the address is not in the wallet (like below)

Reply:
Added